cisco_aci/datadog_checks/cisco_aci/models.py

```diff
@@ -0,0 +1,161 @@
+
+    class EthpmPhysIfAttributes(BaseModel):
+        oper_st: Optional[str] = Field(default=None, alias="operSt")
+        oper_router_mac: Optional[str] = Field(default=None, alias="operRouterMac")
+
+    class EthpmPhysIf(BaseModel):
+        attributes: EthpmPhysIfAttributes
+
+    class L1PhysIfAttributes(BaseModel):
+        admin_st: Optional[str] = Field(default=None, alias="adminSt")
+        id: Optional[str] = None
+        name: Optional[str] = None
+        desc: Optional[str] = None
+        router_mac: Optional[str] = Field(default=None, alias="routerMac")
+
+    class PhysIf(BaseModel):
+        attributes: L1PhysIfAttributes
+        children: Optional[list] = Field(default_factory=list)
+
+        @computed_field
+        @property
+        def ethpm_phys_if(self) -> Optional[EthpmPhysIf]:
+            for child in self.children:
+                if 'ethpmPhysIf' in child:
+                    return EthpmPhysIf(**child['ethpmPhysIf'])
+            return None
+
+    class DeviceMetadata(BaseModel):
+        id: Optional[str] = Field(default=None)
+        id_tags: list = Field(default_factory=list)
+        tags: list = Field(default_factory=list)
+        name: Optional[str] = Field(default=None)
+        ip_address: Optional[str] = Field(default=None)
+        model: Optional[str] = Field(default=None)
+        fabric_st: Optional[str] = Field(default=None, exclude=True)
+        vendor: Optional[str] = Field(default=None)
+        version: Optional[str] = Field(default=None)
+        serial_number: Optional[str] = Field(default=None)
+        device_type: Optional[str] = Field(default=None)
+        integration: Optional[str] = Field(default='cisco-aci')
+
+        @computed_field
+        @property
+        def status(self) -> int:
+            mapping = {
+                'active': 1,
+                'inactive': 2,
+                'disabled': 2,
+                'discovering': 2,
+                'undiscovered': 2,
+                'unsupported': 2,
+                'unknown': 2,
+            }
+            return mapping.get(self.fabric_st, 2)
+
+    class DeviceMetadataList(BaseModel):
+        device_metadata: list = Field(default_factory=list)
+
+    class AdminStatus(IntEnum):
+        UP = 1
+        DOWN = 2
+
+    class OperStatus(IntEnum):
+        UP = 1
+        DOWN = 2
+
+    class Status(str, Enum):
+        UP = "up"
+        DOWN = "down"
+        WARNING = "warning"
+        OFF = "off"
+
+    class InterfaceMetadata(BaseModel):
+        device_id: Optional[str] = Field(default=None)
+        id_tags: list = Field(default_factory=list)
+        index: Optional[str] = Field(default=None)
+        name: Optional[str] = Field(default=None)
+        description: Optional[str] = Field(default=None)
+        mac_address: Optional[str] = Field(default=None)
+        admin_status: Optional[AdminStatus] = Field(default=None)
+        oper_status: Optional[OperStatus] = Field(default=None)
```

Comment:
I just noticed we have an `integration` field for interface metadata too (see [here](https://github.com/DataDog/dd-go/blob/aec04ff24ccbe5d6da6efcead7123caf27c5dd16/apps/network-devices-writer/payload/payload.go#L94))
Probably worth adding it here (i'll also have to add it to cisco sdwan 😅)